vic/vic_run/src/lakes.eb.c

```diff
@@ -290,15 +290,16 @@ solve_lake(double           snowfall,
         *  6. Calculate initial energy balance over ice.
         **********************************************************************/
 
+        windi = (wind * log((2. + soil_con.snow_rough) / soil_con.snow_rough) /
+                 log(wind_h / soil_con.snow_rough));
+        if (windi < 1.0) {
+            windi = 1.0;
+        }
+
```

Comment:
Ahhh, OK, the bug is not in computation of windi.  The bug is that aero_resist should not be computed using windi if there's no ice present.

To be technically correct, we should do the following:
1. leave windi in the `if` block as it was
2. set `windi` to 0 in the `else